src/style/core.ts

```diff
@@ -65,9 +65,8 @@ const root = css`
       color-scheme: var(--rdg-color-scheme, light dark);
 
       /* https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context */
-      /* We set a stacking context so internal elements don't render on top of external components. */
+      /* We set a stacking context so internal elements don't render on top of external elements. */
       contain: strict;
-      contain: size layout style paint;
```

Comment:
I've removed this line, as nowadays both Firefox and Safari support `style` containment, and `strict` is the same as listing all 4 of these.
https://caniuse.com/mdn-css_properties_contain_style
https://developer.mozilla.org/en-US/docs/Web/CSS/contain